uv.schema.json

```diff
@@ -407,11 +407,14 @@
     },
     "publish-url": {
       "description": "The URL for publishing packages to the Python package index (by default: <https://upload.pypi.org/legacy/>).",
-      "type": [
-        "string",
-        "null"
-      ],
-      "format": "uri"
+      "anyOf": [
+        {
+          "$ref": "#/definitions/LogSafeUrl"
+        },
+        {
+          "type": "null"
+        }
+      ]
```

Comment:
The JSON schema here should be just URL (or `uri`), the `LogSafeUrl` description should not be part of the IDE completion help.